source/content/wordpress-wp-options-autoload-cache.md

```diff
@@ -0,0 +1,47 @@
+---
+title: WordPress wp_options Table Autoloading Cache
```

Comment:
We might need to think of a good title in think article something like:

Check your wp_options Table with unnecessary autoloaded data